### PR TITLE
man: add Spanish translation

### PR DESCRIPTION
**Pull request by i-penr**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/man.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# man
+
+> Da formato y muestra páginas del manual.
+> Más información: <https://www.man7.org/linux/man-pages/man1/man.1.html>.
+
+- Mostrar la página del manual para un comando:
```
Comment (navarroaxel) [inline pages.es/linux/man.md:6]: For Romance languages, the present indicative fits better.
```suggestion
- Muestra la página del manual para un comando:
```

File: pages.es/linux/man.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`man {{command}}`
```
Comment (navarroaxel) [inline pages.es/linux/man.md:8]: Please translate the placeholders for get this easier for non-English speakers.
```suggestion
`man {{comando}}`
```

File: pages.es/linux/man.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`man --where {{command}}`
+
+- Muestra la página del manual usando una ubicación específica:
+
+`man --locale={{locale}} {{command}}`
+
+- Busca las páginas del manual que contienen la string indicada:
+
+`man --apropos "{{search_string}}"`
```
Comment (navarroaxel) [inline pages.es/linux/man.md:32]:
```suggestion
`man --apropos "{{cadena_a_buscar}}"`
```

**Comment by navarroaxel** [comment]
Hi @i-penr, please sign the license to pass the CLA check

**Comment by i-penr** [comment]
Sorry about that, I thought placeholders weren't supposed to be changed.

File: pages.es/linux/man.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+> Da formato y muestra páginas del manual.
+> Más información: <https://www.man7.org/linux/man-pages/man1/man.1.html>.
+
+- Muestra la página del manual para un comando:
+
+`man {{comando}}`
+
+- Mostrar la página del manual para un comando de la sección 7:
+
+`man {{7}} {{command}}`
```
Comment (kbdharun) [inline pages.es/linux/man.md:12]:
```suggestion
`man {{7}} {{comando}}`
```

File: pages.es/linux/man.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Listar todas las secciones disponibles para un comando:
+
+`man --whatis {{command}}`
```
Comment (kbdharun) [inline pages.es/linux/man.md:16]:
```suggestion
`man --whatis {{comando}}`
```

File: pages.es/linux/man.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Muestra las rutas usadas para la búsqueda de las páginas:
+
+`man --path`
+
+- Muestra la ubicación de la página del manual en lugar de la propia página:
+
+`man --where {{command}}`
```
Comment (kbdharun) [inline pages.es/linux/man.md:24]:
```suggestion
`man --where {{comando}}`
```

File: pages.es/linux/man.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+- Muestra la página del manual usando una ubicación específica:
+
+`man --locale={{locale}} {{command}}`
```
Comment (kbdharun) [inline pages.es/linux/man.md:28]:
```suggestion
`man --locale={{locale}} {{comando}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by i-penr** [comment]
I was just about to translate the remaining placeholders. Thank you @navarroaxel and @kbdharun for the help, I'll take everything you mentioned into account in future translations. I think everything is done now.

**Review by navarroaxel — APPROVED** [review]
LGTM, we use this new link for man pages.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**